src/daemon/abrt-action-save-package-data.c

```diff
@@ -283,8 +290,12 @@ static int SavePackageDescriptionToDebugDump(const char *dump_dir_name, const ch
 
         char *unpackaged_msg = xasprintf("Crash in unpackaged executable '%s'", executable);
         dd_save_text(dd, FILENAME_UNPACKAGED, unpackaged_msg);
-
         free(unpackaged_msg);
+
+        char *unpackaged_component = xasprintf("%s-unpackaged", basename);
+        dd_save_text(dd, FILENAME_COMPONENT, unpackaged_component);
```

Comment:
Please, do not create the file `component` with artificial names - that would lead to a confusion. Have you checked if there are EVENT conditions with `COMPONENT!=`?

Reply:
We are doing this, because we want to have FAF, which accept reports from unpackaged code. In this case, in FAF, we want to create new component when such report arrives.
On FAF, we need to somehow distinguish between report for which we just do not have downloaded rpm file and between report which was never packaged and we need to create an artificial component in FAF. For that we decided to took basename of that $0.

Reply:
Alright then. So can you create the artificial component on FAF when it receives an unpackaged uReport? IMHO it would be a less error-prone approach because storing this information in a dump directory can confuse other ABRT tools.

Reply:
We can do that easily in FAF. But I believe that we need to edit also satyr(? or other component) where uReport is created, since it requires component to be present.
Maybe do not create new element in dd, but when uReport is created and `component` element is missing, then create an artificial one?